src/embeddings/embeddings_api.cpp

```diff
@@ -31,9 +31,17 @@
 
 using namespace rapidjson;
 
+enum class InputType {
```

Comment:
I just realized that none of those classes are part of ovms namespace. Please do that. When you change it, the InputType will be in ovms namespace which is not good idea either. I think the name is so generic that it could be under EmbeddingsRequest namespace. What's more, I don't think we need to expose it to the API user, it can be hidden (private) within EmbeddingsRequest scope since it is user by this class only anyway.